overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -47333,6 +47430,24 @@
             }
           },
           "taskID:8": "bq_standard:fluid"
+        },
+        "1:10": {
+          "autoConsume:1": 0,
+          "consume:1": 0,
+          "entryLogic:8": "AND",
```

Comment:
Just replace the original task, foil makes much more sense than fluid anyways. Make sure you change the task type to retrieval as well.